tests/cpu/fileformats/FileFormatSpi3D_tests.cpp

```diff
@@ -140,6 +140,24 @@ OCIO_ADD_TEST(FileFormatSpi3D, read_failure)
                               OCIO::Exception,
                               "that falls outside of the cube");
     }
+    {
+        // Duplicated indices
+        const std::string SAMPLE_ERROR =
+            "SPILUT 1.0\n"
+            "3 3\n"
+            "2 2 2\n"
+            "0 0 0 0.0 0.0 0.0\n"
+            "0 0 1 0.0 0.0 0.9\n"
+            "0 0 1 0.0 0.0 0.9\n"
+            "0 1 1 0.0 0.8 0.8\n"
+            "1 0 1 0.7 0.6 0.1\n"
+            "1 1 0 0.6 0.7 0.1\n"
+            "1 1 1 0.6 0.7 0.7\n";
```

Comment:
This LUT should have 8 rows so that we're only testing for duplicates. Currently this would also fail for not enough entries.